packages/test/test-end-to-end-tests/src/test/refreshSerializerLifeCycle.spec.ts

```diff
@@ -271,9 +221,18 @@ describeCompat("Refresh snapshot lifecycle", "NoCompat", (getTestObjectProvider,
 			// container loaded from previous pending state. The snapshot should refresh
 			// in case a summary has already happened. Such snapshot could be the first one to
 			// have a data store with groupId
-			const container2: IContainerExperimental = testConfig.loadOffline
-				? await loadOffline(testContainerConfig, provider, { url }, pendingOps)
-				: await loader.resolve({ url }, pendingOps);
+			let container2: IContainerExperimental;
+			if (testConfig.loadOffline) {
+				const offlineObject = await loadOffline(
+					testContainerConfig,
+					provider,
+					{ url },
+					pendingOps,
+				);
+				container2 = offlineObject.container;
```

Comment:
This avoids having to name `offlineObject` variable
```suggestion
				const { container } = await loadOffline(
					testContainerConfig,
					provider,
					{ url },
					pendingOps,
				);
				container2 = container;
```